pkg/openstack/cinder.go

```diff
@@ -41,6 +41,20 @@ func ReconcileCinder(ctx context.Context, instance *corev1beta1.OpenStackControl
 			//cinder.Spec.DatabaseInstance = instance.Name // name of MariaDB we create here
 			cinder.Spec.DatabaseInstance = "openstack" //FIXME: see above
 		}
+		// if already defined in the top-level crd, we don't override
+		if len(cinder.Spec.ExtraMounts) == 0 {
```

Comment:
wouldn't it better to merge cinder.Spec.ExtraMounts and instance.Spec.ExtraMounts, so that you can have global ones and per service extra volumes? if there are duplicate entries the service one should probably overwrite the global.

Reply:
or have  a parameter to control if those should be merged

Reply:
see comment above